generator/views/glean_ping_view.py

```diff
@@ -87,7 +87,7 @@ def to_lookml(self, bq_client, v1_name: Optional[str]) -> Dict[str, Any]:
                         {
                             "name": "document_id",
                             "type": "string",
-                            "sql": f"${{{self.name}.document_id}}",
+                            "sql": f"${{{self.name}.document_id}}-${{label}}",
```

Comment:
I think we might want to keep `document_id` around as is and create a new dimension with this unique value. Otherwise this might cause confusion if the `document_id` has a different format here.